modules/smtp/setup.php

```diff
@@ -18,6 +18,7 @@
 add_output('compose', 'compose_form_end', true, 'smtp', 'compose_form_content', 'after');
 add_output('compose', 'compose_form_attach', true, 'smtp', 'compose_form_end', 'after');
 add_handler('compose', 'load_smtp_is_imap_forward', true, 'smtp', 'load_user_data', 'after');
+add_handler('compose', 'load_smtp_is_imap_forward_as_attachment', true, 'smtp', 'load_user_data', 'after');
```

Comment:
I'd suggest you do a 'before' insert before the is_imap_forward method or just switch these 2 places - execute forward_as_attachment after load_user_data and then forward after forward_as_attachment. We need to make sure forward_as_attachment is executed before the main forward method.